src/store/modules/industry.module.ts

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios'
+import { GET_INDUSTRIES, NEW_INDUSTRY, DELETE_INDUSTRY, STORE_INDUSTRY } from './actions.type'
+import { SET_INDUSTRIES,ADD_INDUSTRIES, REMOVE_INDUSTRY, SAVE_INDUSTRY } from './mutations.type'
+import { computed } from 'vue'
+
+export const industry_module = {
+    state: {
+        industries: []
+    },
+    actions: {
+        [GET_INDUSTRIES] (context, state){
+            axios.get('http://127.0.0.1:8000/api/admin/industries').then((response) => {
+                context.commit(SET_INDUSTRIES,response)
+            })
+        },
+        [STORE_INDUSTRY] (context, industry){
+            context.commit(SAVE_INDUSTRY, industry.data)
+        },
+        [NEW_INDUSTRY] (context, industry){
+            context.commit(ADD_INDUSTRIES, industry)
+        },
+        [DELETE_INDUSTRY] (context, {industry, index}){
+            context.commit(REMOVE_INDUSTRY, {industry, index})
+        }
+    },
+    mutations: {
+        [SET_INDUSTRIES](state, response){
+            state.industries = response.data.data.items
+            
+        },
+        [ADD_INDUSTRIES] (state, data){
+            state.industries.push(data)
+        },
+        [SAVE_INDUSTRY] (state, industry){
```

Comment:
Any work that is related to axios or using API, must be in actions instead of mutations. mutation should only mutate the state. Here, we are doing axios stuffs, so refactor these methods into actions.